source/kubernetes/versions.rst

```diff
@@ -132,5 +132,12 @@ CNI Version
 ***********
 
 The only supported CNI on Catalyst Cloud Kubernetes Service is Calico and the
-current versions is v3.13.1
+current versions is v3.13.1.
+
+***************
+Addons Versions
+***************
+
+At this stage, Catalyst Cloud Kubernetes Service doesn't support upgrade the
+addons' versions, such as Calico, CoreDNS etc. 
```

Comment:
Is it worth mentioning that these components *will* change when you upgrade a cluster to a newer template version, if the version labels have been updated since the cluster was created?

In a sense you do support upgrading these versions, but only to the single supported version and only when the user changes cluster templates. You don't support using a *custom* version, though.

Reply:
Did you test that since technically I don't think we support it currently because when doing an upgrade, Magnum will only extract the kube_tag label and the ostree_commit and ostree_remote. All the other labels will be ignored. I will test this again to make sure.

Reply:
Oh, that's good to know, that was not my understanding. If that's the case then this is totally fine!